CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultActivityChains.uc

```diff
@@ -103,8 +102,9 @@ static function X2DataTemplate CreateSupplyRaidTemplate()
 	Template.NumInDeck = 1;
 	Template.DeckReq = IsAdvancedChainAvailable;
 	
-	Template.Stages.AddItem('Activity_CommanderSupply');
-	Template.Stages.AddItem('Activity_SupplyRaid');
+	Template.Stages.AddItem(ConstructRandomStage(eActivityType_Infiltration, 'Tag_Intelligence', 'Tag_Distraction'));
+	Template.Stages.AddItem(ConstructPresetStage('Activity_WaitGeneric'));
+	Template.Stages.AddItem(ConstructPresetStage('Activity_SupplyConvoy'));
```

Comment:
Could you explain this change? I mean, the introduction of `Activity_WaitGeneric`

Reply:
It's just a simple wait activity that I can slap inbetween an infil and an assault. Because we don't want infil activities to be immediately followed up by assaults, that's two missions that happen in the same exact hour if we go by game time.